### PR TITLE
Switch guides over to OIDC lieutenant login

### PR DESCRIPTION
**Pull request by bastjan**
I replaced `${COMMODORE_API_TOKEN}` with `$(commodore fetch-token)` to possibly allow for refresh token support.

The guide should be mostly backwards compatible if `commodore fetch-token)` returns the api token.

### CODE REVIEW

**Comment by simu** [comment]
PR which implements `commodore fetch-token`: https://github.com/projectsyn/commodore/pull/475

File: docs/modules/ROOT/partials/recovery/recover-etcd.adoc [hunk 1]
```diff
@@ -50,18 +50,18 @@ The following CLI utilities need to be available locally:
 # For example: https://api.syn.vshn.net
 # IMPORTANT: do NOT add a trailing `/`. Commands below will fail.
 export COMMODORE_API_URL=<lieutenant-api-endpoint>
-export COMMODORE_API_TOKEN=<lieutenant-api-token>
+commodore login
```
Comment (simu) [inline docs/modules/ROOT/partials/recovery/recover-etcd.adoc:53]: not needed

File: docs/modules/ROOT/partials/vshn-input.adoc [hunk 2]
```diff
@@ -8,11 +8,11 @@ export GITLAB_USER=<gitlab-user-name>
 # For example: https://api.syn.vshn.net
 # IMPORTANT: do NOT add a trailing `/`. Commands below will fail.
 export COMMODORE_API_URL=<lieutenant-api-endpoint>
-export COMMODORE_API_TOKEN=<lieutenant-api-token>
+commodore login
```
Comment (simu) [inline docs/modules/ROOT/partials/vshn-input.adoc:11]: not needed

**Review by simu — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/exoscale/decommission.adoc [hunk 3]
```diff
@@ -44,7 +44,7 @@ export GITLAB_USER=<gitlab-user-name>
 # For example: https://api.syn.vshn.net
 # IMPORTANT: do NOT add a trailing `/`. Commands below will fail.
 export COMMODORE_API_URL=<lieutenant-api-endpoint>
-export COMMODORE_API_TOKEN=<lieutenant-api-token>
+commodore login
```
[the remark this answers is not in the document]
Comment (bastjan) [reply docs/modules/ROOT/pages/how-tos/exoscale/decommission.adoc:47]: 👍

**Review by glrf — APPROVED** [review]

**Pull request merged**